README.md

```diff
@@ -164,6 +165,7 @@ cd haskell-language-server
 
 #### Building
 
+
```

Comment:
Why a random blank line here?

Reply:
It's probably inserted by the markdown VSCode plugin